api-spec/extensions/README.md

```diff
@@ -11,8 +11,9 @@ API Extensions given follow the same guidelines for Extension Maturity as given
 
 | Extension Name | Description | Maturity |
 | -------------  | ----------- | -------- |
-| [Fields](fields/README.md) | Adds parameter to constrol which fields are returned in the response. | *Pilot* |
+| [Fields](fields/README.md) | Adds parameter to control which fields are returned in the response. | *Pilot* |
 | [Query](query/README.md) | Adds parameter to search Item and Collection properties. | *Pilot* |
+| [Search](search/README.md) | [ItemCollection](../item-spec/README.md) | Adds search-related metadata to [ItemCollection](../item-spec/README.md). | *Proposal* |
```

Comment:
Could you link directly to the ItemCollection section?

Reply:
fixed the link and removed the extra column that came from the table in the core extensions that i copied this from.